chat-android/src/test/java/com/ably/chat/integration/MessagesIntegrationTest.kt

```diff
@@ -248,6 +250,17 @@ class MessagesIntegrationTest {
         assertEquals(deletedMessage.action, receivedMsg2.action)
     }
 
+    @Test
+    fun `messages channel should include agent channel param`() = runTest {
+        val chatClient = sandbox.createSandboxChatClient()
+        val roomId = UUID.randomUUID().toString()
+        val room = chatClient.rooms.get(roomId)
+        assertEquals(
+            "chat-kotlin/${BuildConfig.APP_VERSION}",
+            room.messages.channel.channelOptions?.params?.get("agent"),
```

Comment:
is this set via `wrappedChannel`, because we have deleted explicit code for this right?

Reply:
yep, now `createWrapperSdkProxy` responsible for adding agent information